tmvis.js

```diff
@@ -834,6 +834,10 @@ function getTimemapGodFunctionForAlSummarization (uri, response,curCookieClientI
   else if(response.thumbnails['primesource']=="internetarchive"){
       timemapHost = 'web.archive.org'
       timemapPath = '/web/timemap/link/' + uri
+      if(uri.search('/web/timemap/link/') > 0)
```

Comment:
Unless you plan to use RegExp here, `indexOf()` would be a better alternative to the `search()` function.

```suggestion
      if(uri.indexOf('/web/timemap/link/') > 0)
```